src/libcore/ops.rs

```diff
@@ -102,6 +102,12 @@ pub trait Drop {
     ///
     /// After this function is over, the memory of `self` will be deallocated.
     ///
+    /// This function cannot be called explicitly. This is compiler error
+    /// [0040](https://doc.rust-lang.org/error-index.html#E0040). However, the
```

Comment:
@GuillaumeGomez it would be nice of you to mention RFC you’re referring to. The contributor seems to be submitting for the first time, so you cannot expect them to know what you mean by just an RFC.

@matthew-piziak The RFC in question is [this one](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md). It documents some conventions which are to be used when writing documentation. The section pertaining to link usage is at the bottom of [this section](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#using-markdown-1).